shared/src/business/entities/judgeActivityReport/JudgeActivityReportSearch.ts

```diff
@@ -18,6 +22,11 @@ export class JudgeActivityReportSearch extends JoiValidationEntity {
   public startDate: string;
   public judgeName: string;
   public judgeId: string;
+  public statuses: CAV_AND_SUBMITTED_CASE_STATUS_TYPES;
+  public judges: string[];
```

Comment:
Are both `judgeId` and `judges` needed to run this report now? Is there anyway to unify them into one property?